src/NLog.Web/README.md

```diff
@@ -0,0 +1,15 @@
+# NLog.Web
+
+- LoggerProvider for ASP.NET platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
```

Comment:
All .NET Framework versions are supported as you can see here: https://www.nuget.org/packages/NLog.Web#supportedframeworks-body-tab

Reply:
changed